### PR TITLE
Fixed incorrect link in the blog of 10 Years of Kubernetes

### PR DESCRIPTION
**Pull request by mengjiao-liu**
The link points to a blog that does not match the content.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Some notable updates, milestones and events since 1.0 include:
 * December 2017 — [Kubernetes 1.9](/blog/2017/12/kubernetes-19-workloads-expanded-ecosystem/) sees the Workloads API becoming GA (Generally Available). The release blog states: _"Deployment and ReplicaSet, two of the most commonly used objects in Kubernetes, are now stabilized after more than a year of real-world use and feedback."_
 * December 2018 — In 1.13, the Container Storage Interface (CSI) reaches GA, kubeadm tool for bootstrapping minimum viable clusters reaches GA, and CoreDNS becomes the default DNS server.
 * September 2019 — [Custom Resource Definitions go GA](/blog/2019/09/18/kubernetes-1-16-release-announcement/) in Kubernetes 1.16.
-* August 2020 — [Kubernetes 1.19](/blog/2016/12/kubernetes-1-5-supporting-production-workloads/) increases the support window for releases to 1 year.
+* August 2020 — [Kubernetes 1.19](/blog/2020/08/31/kubernetes-1-19-feature-one-year-support/) increases the support window for releases to 1 year.
```
Comment (Ritikaa96) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:110]: @sftim @mengjiao-liu shouldn't we have written it as per the previous points. i.e

[Increases the support window for releases to 1 year](/blog/2020/08/31/kubernetes-1-19-feature-one-year-support/) in Kubernetes 1.19

Comment (Ritikaa96) [reply content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:110]: as the link is not directly to the release blog but regarding the topic , also it is consistent with previous points. 
Also we could add a link for v1.13 point.
I know this PR has been merged but just wanted to mention

Comment (mengjiao-liu) [reply content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:110]: Thanks for your suggestion! 
There are two options:
1.  the link is directly to the topic
[Increases the support window for releases to 1 year](https://github.com/blog/2020/08/31/kubernetes-1-19-feature-one-year-support/) in Kubernetes 1.19
2.  the link is directly to the  Kubernetes 1.19 release blog: https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/
    August 2020 — [Kubernetes 1.19](/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/) increases the support window for releases to 1 year.

WDYT? @sftim

**Pull request merged**